### DISCUSSION TITLE
GUI Tool?

### DISCUSSION THREAD

**Discussion by dandy83ten** [axmolengine/axmol#1344] [Q&A] [answered]
Is there a UI Tool for axmol?
Do you have any  plans to develop UI tool?

**Comment by iAndyHD3** [comment]
gui for what?

**Comment by dandy83ten** [comment]
UI editor
I want to work on the UI easily.

**Reply by aismann** [reply, marked as answer]
https://www.fairygui.com/

**Reply by halx99** [reply]
Yes fairygui is a choice for UI easily, there is a online demo: https://axmol.netlify.app/wasm/fairygui-tests/fairygui-tests

**Reply by iAbdulQadirKhan** [reply]
How can I get fairygui_tests code? 
By this link I can play the test online. [https://axmol.netlify.app/wasm/fairygui-tests/fairygui-tests](url)

**Reply by iAndyHD3** [reply]
https://github.com/axmolengine/axmol/tree/dev/tests/fairygui-tests
